kms/decrypt_rsa.go

```diff
@@ -24,9 +24,12 @@ import (
 	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
 )
 
-// decryptRSA will attempt to decrypt a given ciphertext with an 'RSA_DECRYPT_OAEP_2048_SHA256' private key.stored on Cloud KMS
+// decryptRSA will attempt to decrypt a given ciphertext with an 'RSA_DECRYPT_OAEP_2048_SHA256'
+// private key stored on Cloud KMS.
 func decryptRSA(name string, ciphertext []byte) ([]byte, error) {
 	// name := "projects/PROJECT_ID/locations/global/keyRings/RING_ID/cryptoKeys/KEY_ID/cryptoKeyVersions/1"
+	// cipherBytes, err := encryptRSA(rsaDecryptPath, []byte("Sample message"))
```

Comment:
It's in a comment, but check this error?

Reply:
Done